Assets/AltTester/Runtime/UI/AltDialog.cs

```diff
@@ -33,6 +33,9 @@ public class AltDialog : UnityEngine.MonoBehaviour
         private readonly UnityEngine.Color SUCCESS_COLOR = new UnityEngine.Color32(0, 165, 36, 255);
         private readonly UnityEngine.Color WARNING_COLOR = new UnityEngine.Color32(255, 255, 95, 255);
         private readonly UnityEngine.Color ERROR_COLOR = new UnityEngine.Color32(191, 71, 85, 255);
+        private readonly string HOST = "host";
+        private readonly string PORT = "port";
+        private readonly string APP_NAME = "appName";
```

Comment:
Please make this value less generic in order to not overwrite if also the client users use this